docs/samples/browser-plugin-meetings/app.js

```diff
@@ -625,15 +625,19 @@ async function joinMeeting({withMedia, withDevice} = {withMedia: false, withDevi
     meeting.on('meeting:stoppedSharingRemote', () => {
       /**
        * Here we first remove the media to stop showing the remote stream being received and again
-       * attach the remote media stream (which gets added during addMedia() call) but since the remote 
-       * is not sending the frames anymore it does not show the screenshare but keeps the event attached. 
+       * attach the remote media stream (which gets added during addMedia() call) but since the remote
+       * is not sending the frames anymore it does not show the screenshare but keeps the event attached.
        * Hence, when we again start sharing the screen, the media flows correctly to the video element.
        */
       const remoteShareTemp = meetingStreamsRemoteShare.srcObject;
       meetingStreamsRemoteShare.srcObject = null;
       meetingStreamsRemoteShare.srcObject = remoteShareTemp;
     });
 
+    meeting.on('meeting:self:mutedByOthers', () => {
```

Comment:
what about "meeting:self:unmutedByOthers"?

Reply:
We don't have any action to be done in Samples app associated with this event hence not listening for it here.

Reply:
don't we need to update the UI, like labels on the button?

Reply:
My understanding here is that Unmuting by others is not possible so no update would be required.

Reply:
it is possible for certain meetings, you can configure it: 
![image](https://github.com/webex/webex-js-sdk/assets/7473332/aa396d9d-1e9f-4395-9125-18ada38e6fb0)

Reply:
addressed..added the event now